packages/volto/src/components/manage/UniversalLink/UniversalLink.jsx

```diff
@@ -8,10 +8,11 @@ import PropTypes from 'prop-types';
 import { HashLink as Link } from 'react-router-hash-link';
 import { useSelector } from 'react-redux';
 import {
+  addPrefixPath,
   flattenToAppURL,
   isInternalURL,
   URLUtils,
-} from '@plone/volto/helpers/Url/Url';
```

Comment:
We decided not using barrel indexes in Volto anymore, so going back here is not an option.